cwl_wes/custom_config.py

```diff
@@ -3,22 +3,107 @@
 from typing import Dict, List, Optional
 from foca.models.config import FOCABaseConfig
 
+
 class StorageConfig(FOCABaseConfig):
+    """Model for task run and storage configuration.
+
+    Args:
+        tmp_dir: Temporary run directory path
+        permanent_dir: Permanent working directory path
+        remote_storage_url: Remote file storage FTP endpoint
+
+    Attributes:
+        tmp_dir: Temporary run directory path
+        permanent_dir: Permanent working directory path
+        remote_storage_url: Remote file storage FTP endpoint
+
+    Raises:
+        pydantic.ValidationError: The class was instantianted with an illegal
+            data type.
```

Comment:
I don't think we need that as it is not the model class _itself_ that raises this error (or any error, in fact)